README.md

```diff
@@ -1,25 +1,35 @@
 # scraper
 
-This repository contains our scraping code. 
-It pulls companies from [/r/cscareerquestions](https://reddit.com/r/cscareerquestions) and crawls the web for positions 
-relating to those companies. We are also actively looking for other reliable sources of company names. 
+This repository contains our scraping code.
+The scraper pulls companies from [/r/cscareerquestions](https://reddit.com/r/cscareerquestions)
+and crawls the web for positions relating to those companies.
+We are also actively looking for other reliable sources
+of company names.
 
 ## Installation
 
 Make sure the following dependencies have been installed on your system.
 
-* Google Chrome/Chromium v74
-* Java 8
-* Gradle
+* Docker
 
-Afterwards, clone this repository and `cd` in.
+You will also need to acquire a valid **hibernate.cfg.xml** file and place it
```

Comment:
Link to example one don't just say acquire

Reply:
I added an example in the resources directory that people can refer to.